trace-obfuscation/src/obfuscation_config.rs

```diff
@@ -0,0 +1,44 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2023-Present Datadog, Inc.
+
+use log::{debug, error};
+use std::env;
+
+use ddcommon::config::parse_env;
+
+use crate::replacer::{self, ReplaceRule};
+
+#[derive(Debug)]
+pub struct ObfuscationConfig {
+    pub tag_replace_rules: Option<Vec<ReplaceRule>>,
+    pub http_remove_query_string: bool,
+    pub http_remove_path_digits: bool,
+}
+
+impl ObfuscationConfig {
+    pub fn new() -> Result<ObfuscationConfig, Box<dyn std::error::Error>> {
+        let tag_replace_rules: Option<Vec<ReplaceRule>> = match env::var("DD_APM_REPLACE_TAGS") {
```

Comment:
are we supporting config from datadog.yaml file?

Reply:
not currently, no. just from env vars